apps/dotcom/client/src/tla/app/TldrawApp.ts

```diff
@@ -104,6 +106,11 @@ export class TldrawApp {
 		)
 	}
 
+	private getIntl() {
+		const locale = this.user$.get()?.locale
+		return createIntl({ locale: locale ?? 'en' })
```

Comment:
@mimecuvalo looks like this is the way to use react intl outside of react?

Reply:
yes, correct. however, slightly annoying thing is you should add a bit of caching by calling `setupCreateIntl` (see in `i18n.tsx`). and you'll need to update that cache when the locale changes.

as long as you're looking at this can you update the `setupCreateIntl` comment to the correct link? i see i put an old one in there, should be https://formatjs.github.io/docs/react-intl/api/#createintl